swatch-core/src/main/resources/tag_profile.yaml

```diff
@@ -222,6 +222,17 @@ tagMetrics:
       prometheusMetric: kafka_id:strimzi_resource_state:max_over_time1h
       prometheusMetadataMetric: subscription_labels
 
+  # ODF metrics
+  - tag: Data-Foundation-metrics
+    metricId: redhat.com:openshift_container_platform:cpu_hour
+    uom: CORES
+    queryKey: 5mSamples
+    queryParams:
+      product: ocp
+      # This promQL will list cpu cores (as counted by OCP) for clusters that have the ocs-operator installed (part of ODF)
+      prometheusMetric: (subscription_sync_total{name="ocs-operator"})*0+ on(_id) group_left(value)(max by (_id) (cluster:usage:workload:capacity_physical_cpu_cores:max:5m))
```

Comment:
If you want a single metric, then it would be `cluster:usage:workload:capacity_physical_cpu_cores:max:5m` the reason why I did not leave it as that is that we do not want to charge customers _until_ they actually at least installed the operator (which we'll get through the left-join with `subscription_sync_total{name="ocs-operator"}`